src/components/transfers/custom/FormSelect.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { useField, useFormikContext } from 'formik'
+import { useTranslation } from 'next-i18next'
+
+import { TextField, TextFieldProps } from '@mui/material'
+
+import { translateError } from 'common/form/useForm'
+import { TranslatableField } from 'common/form/validation'
+
+export type selectProps = {
+  label: string
+  name: string
+} & TextFieldProps
+
+export default function FormSelect({ label, name, ...textFieldProps }: selectProps) {
```

Comment:
This component needs to be in `components/common` not in `components/transfer`


It's not really clear what's the purpose of this component